tests/gold_tests/tls/tls_client_cert.test.py

```diff
@@ -293,3 +306,21 @@ def done_reload(process, hasRunFor, **kw):
 tr4fail.Processes.Default.Command = 'curl  -H host:example.com http://127.0.0.1:{0}/case2'.format(ts.Variables.port)
 tr4fail.Processes.Default.ReturnCode = 0
 tr4fail.Processes.Default.Streams.stdout = Testers.ContainsExpression("Could Not Connect", "Check response")
+
+# Parking this as a ready tester on a meaningless process
+# Stall the test runs until the squid.log file has appeared
+def access_log_ready(tsenv):
+  def done_access_log(process, hasRunFor, **kw):
+    cmd = "test -f {0}".format(ts.Disk.squid_log)
+    retval = subprocess.run(cmd, shell=True, env=tsenv)
+    return retval.returncode == 0
+
+  return done_access_log
+
+tr = Test.AddTestRun("Wait for the access log to write out")
+tr.Processes.Default.StartBefore(server4, ready=access_log_ready(ts.Env))
+tr.StillRunningAfter = ts
+tr.Processes.Default.Command = 'ls'
```

Comment:
Nitpick:  I'd use 'echo "log file exists"' or such instead of 'ls'.

Reply:
Or actually how about 'sleep 1 ; echo "log file ready"' as a failsafe to ensure that TS has finished writing the log file before Au test compares it to the gold file.

Reply:
Given the size of the test and the number of log entries, it seems likely likely that this will get pushed out in one flush.

Reply:
Hmm seems questionable that creating a directory entry and writing the file would be a single atomic filesystem op.  But, since this is Python, I suppose the delay between seeing the file exists and actually reading it should  be sufficient for the writing to be complete.